lib/src/main/kotlin/starknet/data/types/execution.kt

```diff
@@ -0,0 +1,38 @@
+package starknet.data.types
+
+import starknet.data.selectorFromName
+import types.Felt
+
+data class Call(
+    val contractAddress: Felt,
+    val entrypoint: String,
+    val calldata: Calldata,
```

Comment:
Right now in Invocation data class, calldata is optional, do we want it to be optional here as well?

Reply:
Calldata itself is an optional type, right?